### DISCUSSION TITLE
How do I host ArchiveBox on my local network?

### DISCUSSION THREAD

**Discussion by Beck-sys** [ArchiveBox/ArchiveBox#811] [Q&A] [answered]
Not using Docker--with the APT instructions.
My apologies for not knowing how to do this. I've tried to look up resources but they did not make sense to me in the context of ArchiveBox.
I'd like to be able to view ArchiveBox and add things to it from any computer on my network by connecting to the local server from the PC running ArchiveBox

**Comment by pirate** [comment, marked as answer]
https://github.com/ArchiveBox/ArchiveBox#-manual-setup > `apt`

```bash
sudo add-apt-repository -u ppa:archivebox/archivebox
sudo apt install archivebox

mkdir -p ~/archivebox && cd ~/archivebox
archivebox init --setup
archivebox server 0.0.0.0:9595

ip addr | grep 192.168 # find your LAN ip
```

Then visit the ip of your server:9595 from other computers on your network to access archivebox:
`http://<your lan ip>:9595`

**Reply by Beck-sys** [reply]
Thanks so much!!

**Reply by Beck-sys** [reply]
@pirate I am also wondering how to do this for the Docker Compose instructions. I tried doing `archivebox server 0.0.0.0:9595` and it did not recognize the command. So, I assume it would be different for using Docker Compose
Sorry for all of my simple questions!
